scripts/system_performance/models/query_argument_constructor.py

```diff
@@ -0,0 +1,116 @@
+# ############################################################################### #
```

Comment:
This should be renamed. The name `query_argument_constructor` suggests that it's job is to construct the queries and arguments but this is also responsible for running queries as well. As such you could just call this file `model.py` or `database_model.py` if you have various models